src/main/java/com/coronacarecard/controller/StripePaymentController.java

```diff
@@ -57,7 +58,7 @@ public CheckoutResponse fail(String urlParams) {
     @GetMapping("/business/onboard/{id}")
     public String onboard(@PathVariable String id) throws BusinessNotFoundException, InternalException {
         try {
-            Long businessId = Long.parseLong(id);
+            UUID businessId = UUID.fromString(id);
```

Comment:
Why not change the `String id` to UUID like you did it above?

Reply:
Good catch. I am making the switch.